deeplearning4j/deeplearning4j-core/src/test/java/org/deeplearning4j/nn/layers/samediff/testlayers/SameDiffConv.java

```diff
@@ -161,14 +160,51 @@ public void applyGlobalConfigToLayer(NeuralNetConfiguration.Builder globalConfig
 
     public static class Builder extends SameDiffLayer.Builder<Builder> {
 
+        @Getter
+        @Setter
```

Comment:
This applies everywhere in this PR: I think we should be able to use ```@Getter``` and ```@Setter``` on the builder classes, rather than having these two annotations on every single field? That would be a lot cleaner, with the same result I think.